spec/models/booking_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.describe Booking, type: :model do
+  subject { described_class.new }
```

Comment:
Change this to:
 `subject { described_class.new(subject.user_id: 12,
                                                  subject.resource_id: 13,
                                                  .
                                                  . etc. }`

And then only change specific fields for specific tests in order not to multiply the same code.